components/explore/DesktopControls.vue

```diff
@@ -6,14 +6,12 @@
     </div>
 
     <div
-      v-if="!isActivityTab"
+      v-if="isExploreCollections || isExploreItems || isCollection"
       class="explore-menu flex"
     >
       <ExploreSort />
-      <ChainDropdown
-        v-if="!route.name?.toString().includes('prefix-collection-id')"
-      />
-      <GridLayoutControls v-if="!isCollection" />
+      <ChainDropdown v-if="isExploreItems || isExploreCollections" />
```

Comment:
lets add missing required prop here?

Reply:
fixed by making it optional . already had a default valud